src/assets/ElmAsset.js

```diff
@@ -0,0 +1,45 @@
+const JSAsset = require('./JSAsset');
+const localRequire = require('../utils/localRequire');
+const process = require('process');
+const config = require('../utils/config');
+
+class ElmAsset extends JSAsset {
+  async getParserOptions() {
+    let defaultOptions = {
+      yes: true,
+      cwd: process.cwd()
+    };
+
+    // grab other optios from package.elm.json
+    const elmOptions = await config.load(this.name, ['elm-package.json']);
+
+    if (elmOptions.nodeElmCompilerOptions) {
+      defaultOptions = Object.assign(
+        elmOptions.nodeElmCompilerOptions,
+        defaultOptions
+      );
+    }
+
+    // Use a local copy of elm-make when in test
+    if (process.env.NODE_ENV === 'test') {
+      defaultOptions.pathToMake = require.resolve('elm/binwrappers/elm-make');
+    }
+    return defaultOptions;
+  }
+
```

Comment:
I think you'll want to override `getDependencies` here and provide Parcel with a list of Elm deps. I've made a PR for `parcel-plugin-elm` that does just that right here:

https://github.com/ssuman/parcel-plugin-elm/pull/3#issuecomment-357263751